packages/mobile/src/store/products/selectors.ts

```diff
@@ -14,4 +16,21 @@ const selectProductById = createSelector(
   },
 );
 
-export { selectProductById, selectProducts };
+const selectAuctionProducts = (state: RootState): ProductDto[] => {
+  return state.products.products.filter(
+    (item) => item.type === ProductType.AUCTION,
+  );
+};
+
+const selectSellingProducts = (state: RootState): ProductDto[] => {
+  return state.products.products.filter(
+    (item) => item.type === ProductType.SELLING,
+  );
+};
```

Comment:
These two can be combined under one selector:
`const selectProductsByType = createSelector([selectProducts, (state: RootState, productType: ProductType) => productType], (products, type) => products.filter(item => item.type === type))`